client/lib/text-utils/index.js

```diff
@@ -24,4 +30,26 @@ function countWords( content ) {
 	return 0;
 }
 
-export default { countWords };
+const diffWords = JsDiff.diffWords;
+
+function countDiffWords( diffChanges ) {
+	return reduce(
+		diffChanges,
+		( accumulator, change ) => {
+			const count = countWords( change.value );
+			change.added && ( accumulator.added += count );
+			change.removed && ( accumulator.removed += count );
+			return accumulator;
+		},
+		{
+			added: 0,
+			removed: 0,
+		},
+	);
+}
+
+export default {
```

Comment:
Minor and not related to these changes specifically: Exporting only a default of an object makes it difficult to [tree shake](https://webpack.js.org/guides/tree-shaking/) (dead code elimination) unused functions within a file. We're not yet on Webpack 2 (#10799, #11204) so this isn't entirely relevant yet anyways, but in anticipation it's typically better to export/import functions individually.